packages/notifi-react-card/lib/components/subscription/subscription-card-views/PreviewCard.tsx

```diff
@@ -16,6 +18,7 @@ export type PreviewCardProps = Readonly<{
     AlertsPanel?: DeepPartialReadonly<AlertsPanelProps['classNames']>;
     backArrowContainer?: string;
     UserInfoPanel?: DeepPartialReadonly<UserInfoPanelProps['classNames']>;
+    signupBanner?: SignupBannerProps['classNames']; // TODO: Move up one level (for MVP-2733), Blocker: MVP-2716
```

Comment:
same question here

Reply:
Yeah, this is the action for next refactor PR (MVP-2733). 
Now, the The new component `<signupBanner/>`s are placed in both `HistoryCardView` and `PreviewCard` separately. The plan is to move them both up to `SubscriptionCardV1` so that we do not need to pass in the signupBanner the extra layer. 
cc @juni-notifi 

> This action is planning to be done in next Ticket (MVP-2733) because this can be done after both #320 & #321 are merged because 
> 1. These two PRs is supposed to be one feature, so we can not only merge one (major)
> 2. To avoid conflict (minor).